tests/standalone/mongoclient-connection-options-fsync-2.6.phpt

```diff
@@ -47,12 +47,11 @@ dump_writeConcern($mn);
 
 echo "Setting it to false, per-query, and w=0 to force no-gle\n";
 $doc = array("doc" => "ument");
+printLogs(MongoLog::IO, MongoLog::FINE, '/is_gle_op/');
```

Comment:
From the commit message:

> Stream notifications cannot be used to assert the write concern of unacknowledged writes, since they are routed through legacy opcodes instead of write commands. Use MongoLog to assert that no GLE is issued for these writes.